test/specs/HomeWork.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Tc-1:
+ * https://www.darsky.net
+ * Verify feelsLikeTempValue is between lowTempValue and highTempValue
+ */
+
+describe('Homework', () =>{
+    it ('Verify feelsLikeTempValue is between lowTempValue and highTempValue', async () => {
+        await browser.url('https://www.darsky.net/')
+        await browser.pause(3000);
+        const feelIsLikeTemp = $('span[class=feels-like-text]').getText();
+        const lowTemp = $('span[class=low-temp-text]').getText();
+        const highTemp = $('span[class=high-temp-text]').getText();
+        const numfeelIsLikeTemp = Number (feelIsLikeTemp.substring(0, feelIsLikeTemp.lenght - 1));
```

Comment:
correct your code. it should be
feelIsLikeTemp.length 
instead
feelIsLikeTemp.lenght